llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -23877,10 +23821,11 @@ bool RISCVTargetLowering::lowerInterleavedLoad(
   };
 
   Value *VL = ConstantInt::get(XLenTy, VTy->getNumElements());
-
-  CallInst *VlsegN = Builder.CreateIntrinsic(
-      FixedVlsegIntrIds[Factor - 2], {VTy, LI->getPointerOperandType(), XLenTy},
-      {LI->getPointerOperand(), VL});
+  // All-ones mask.
```

Comment:
Nit, I don't think we need the comment since the builder function explains it in the name

Reply:
Fixed